src/database.js

```diff
@@ -0,0 +1,88 @@
+import fs from 'fs';
+import sqlite3 from 'sqlite3';
+
+const schema = fs.readFileSync('./src/schema.sql').toString();
+const connect = new Promise((resolve, reject) => {
+  const result = new sqlite3.Database('./.data/application.db', (error) => {
+    if (error) {
+      reject(error);
+    } else {
+      console.log('i have hereby connected :)')
+      result.exec(schema, (execError) => {
```

Comment:
This connect promise opens a connection to the new database once on boot, then runs the contents of schema.sql, which is assumed to be idempotent.

Reply:
so happy having all the schema specifics out of the .js file!!!!